src/services/form-validation/validateSubmission.ts

```diff
@@ -572,12 +572,20 @@ export default function validateSubmission({
 
           const entries = Array.isArray(value) ? value : []
 
+          const repeatableSetExecutedLookups =
+            executedLookups !== undefined &&
+            typeof executedLookups !== 'boolean' &&
+            !Array.isArray(executedLookups) &&
+            Array.isArray(executedLookups[formElement.name])
+              ? executedLookups[formElement.name]
+              : []
```

Comment:
Was this existing, or a result our implementation?

Reply:
Existing, gross and I would like to clean it up but not today.